CodePush.m

```diff
@@ -56,6 +57,7 @@ + (NSURL *)bundleURLForResource:(NSString *)resourceName
     NSError *error;
     NSString *packageFile = [CodePushPackage getCurrentPackageBundlePath:&error];
     NSURL *binaryJsBundleUrl = [[NSBundle mainBundle] URLForResource:resourceName withExtension:resourceExtension];
+    [self saveBinaryBundleDate:binaryJsBundleUrl];
```

Comment:
Does this really need to happen on every app launch? Maybe it would make more sense to use an ivar to carry this value? The write/synchronize to user defaults is causing disk I/O, so I'm just wondering if that is the best solution for this afterall.